tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+bok_choy==0.3.2
```

Comment:
* the bok-choy requirement is from https://github.com/edx/xblock-sdk/blob/master/test-requirements.txt , and comes if you use the SeleniumTest class in xblock-sdk or xblock-utils, which this block does I believe. However, I would suggest removing almost all the dependencies seen in this file and just depend on xblock-sdk, which specifies the versions we need. I don't like having pinned versions all over the place, which is a pain to maintain and leads to conflicts.
* XBlock has always supported real children - it's only edx-platform that didn't